.github/workflows/ci_cron_weekly.yml

```diff
@@ -145,6 +146,8 @@ jobs:
                                   liberfa1
 
           run: |
+            uname -a
+            getconf LONG_BIT
```

Comment:
These only exist to so we can confirm that some of the jobs are 32 bit.

Reply:
Your comment above is a little incomplete, so I am trying to understand more. Do you mean that `getconf LONG_BIT` would just crash if it is not a true 32-bit machine? Or is it suppose to print something out to the log?

Reply:
This prints out how many bits a floating point is by default in the architecture. It should return 32, which it does.

Reply:
```
2023-07-07T16:50:23.8975207Z   run: uname -a
getconf LONG_BIT
python3 -m venv --system-site-packages tests
source tests/bin/activate
ASTROPY_USE_SYSTEM_ALL=1 pip3 install -e .[test]
python3 -m pytest
```

I don't see any number printed out here.

Reply:
```
uname -a
getconf LONG_BIT
```

Outputs are right here:

- https://github.com/astropy/astropy/actions/runs/5488698114/jobs/10001903172?pr=15018#step:3:1000
- https://github.com/astropy/astropy/actions/runs/5488698114/jobs/10001903172?pr=15018#step:3:1001

Reply:
Anyway, we can make it print out `LONG_BIT=32` instead of just `32`? It is really hard to find.